tests/navjoy_translator_test.py

```diff
@@ -0,0 +1,1065 @@
+from translator import navjoy_translator
+import xmltodict
+from unittest.mock import MagicMock, patch, Mock
+import unittest
+import os
+# from zoneinfo import ZoneInfo
+import time_machine
+import uuid
+from datetime import datetime, timezone
+from freezegun import freeze_time
```

Comment:
`freezegun` doesn't seem to be included in the requirements.txt file and is causing unit tests to fail. It doesn't seem to be used, however and if removed tests succeed.

Reply:
removed this, good catch. Was playing around with mocking datetime.utcnow and that was accidentally left around